tools/bamm-cli/src/test/java/io/openmanufacturing/sds/ApplicationIntegrationTest.java

```diff
@@ -72,28 +83,23 @@ public void beforeEach() throws IOException {
       if ( !outputDirectory.toFile().mkdirs() ) {
          throw new RuntimeException();
       }
+      stdoutBuffer.reset();
    }
 
    @Test
    public void testNoArgs() throws Exception {
-      assertThat( catchSystemExit( bammCli::run ) ).isEqualTo( 1 );
+      assertThat( catchSystemExit( () -> bammCli.run( new String[0] ) ) ).isEqualTo( 0 );
    }
 
    @Test
    public void testWrongArgs() throws Exception {
-      assertThat( catchSystemExit( () -> bammCli.run( "-i", "notexists" ) ) ).isEqualTo( 1 );
+      assertThat( catchSystemExit( () -> bammCli.run( "-i", "notexists" ) ) ).isEqualTo( 2 );
```

Comment:
Why is the expected error code 2?
I would expect one of the following behaviours:
1. Successful run returns 0, failure returns 1
2. Successful run returns 0, failure returns some error code, but all error possible codes are documented